APCAppCore/APCAppCore/UI/TabBarControllers/NewsFeed/APCNewsFeedViewController.m

```diff
@@ -116,7 +116,18 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     APCFeedItem *item = self.posts[indexPath.row];
     
     cell.titleLabel.text = item.title;
-    cell.descriptionLabel.text = item.itemDescription;
+    
+    // strip HTML tags from itemDescription before displaying.
+    NSError *error = nil;
+    NSAttributedString *attributedDescription = [[NSAttributedString alloc] initWithData:[item.itemDescription dataUsingEncoding:NSUTF8StringEncoding] options:@{NSDocumentTypeDocumentAttribute: NSHTMLTextDocumentType} documentAttributes:nil error:&error];
+    if (error) {
+#if DEBUG
+        NSLog(@"Problem creating attributed string from HTML:\n%@\nError:\n%@", item.description, error);
+#endif
```

Comment:
seems like someday we might want to implement one of the approaches here: http://stackoverflow.com/questions/2025471/do-i-need-to-disable-nslog-before-release-application